nlg/README.md

```diff
@@ -0,0 +1,136 @@
+---
+title: Natural Language Generation
+prefix: Natural Language Generation
+...
+
+[TOC]
+
+From v1.54, Gramex includes a Natural Language Generation component. The NLG app
+is a smart template creator which:
+
+1. Automatically creates tornado templates from English text in the context of a dataset.
+2. Allows for modification and generalization of these templates.
+3. Renders these templates as a unified narrative.
+
+To use it, add the following to your `gramex.yaml`:
+
+```yaml
+import:
+  nlg:
+    path: $GRAMEXAPPS/nlg/gramex.yaml
+    YAMLURL: $YAMLURL/nlg/
+    auth:
+      login_url: /$YAMLURL/login
+```
+
+This configuration mounts the app at [nlg/](nlg/):
```

Comment:
This is working against which gramex version? I'm not able to run from gramex dev.

Reply:
This is supposed to run with [this branch of gramex](https://github.com/gramener/gramex/tree/jd-nlg-doc), the PR for which is at gramener/gramex#143